gradle/plugins/src/main/kotlin/com.hedera.gradle.repositories.gradle.kts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+repositories {
+    mavenCentral()
+    maven("https://jitpack.io") {
+        content { includeModule("io.github.cdimascio", "java-dotenv") }
```

Comment:
Please help me to understand why this change was introduced

Reply:
This instructs Gradle to **only** look for `java-dotenv` in the "jitpack.io" repo. And it is documentation for us to remember why we need that repo – in addition to `mavenCentral()` – at all.

But your question makes me realize that we should change the order. Which I did now. Gradle goes through the repositories from top to bottom. Now, it will only look for `java-dotenv` in JitPack (and find it). It will not make any other requests to that repo, because of the the `content { ... }` filter. All other requests will go directly to mavenCentral(). With that, we do not have any unnecessary requests to any of the repositories.